__tests__/app.test.js

```diff
@@ -107,6 +106,58 @@ describe("api/topics", () => {
       })
     });
   });
+  describe.only('api/articles', () => {
+    test.only('should respond with an articles array of article objects with the author set as the username from the users table, sorted in descending order', () => {
+      return request(app)
+      .get("/api/articles?sort_by=created_at&order=desc")
```

Comment:
This is a perfectly valid test as these queries are accepted. However, this should also be the case by default. Don't remove this test, but you can add another where no queries are applied and the sort order is automatically in place